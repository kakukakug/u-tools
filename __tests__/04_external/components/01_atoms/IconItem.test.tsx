// import React from "react";
// import { render, fireEvent } from "@testing-library/react-native";

// import { IconItem } from "../../../../src/04_external/components/01_atoms/IconItem";

describe("IconItem", () => {
  it(`test 実装が必要`, () => {
    expect(true).toBeTruthy();
  });
  // const mockOnPress = jest.fn();
  // const mockName = "star";
  // const mockFamilyName = "Ionicons";
  // it(`IconItemを描写`, () => {
  // const component = render(
  // <IconItem
  // name={mockName}
  // onPress={mockOnPress}
  // selectFamily={mockFamilyName}
  // />
  // );

  // const name = component.getByText(mockName);
  // expect(name).toBeTruthy();

  // fireEvent.press(name);
  // expect(mockOnPress).toBeCalled();

  // expect(component.toJSON()).toMatchSnapshot();
  // });
});
