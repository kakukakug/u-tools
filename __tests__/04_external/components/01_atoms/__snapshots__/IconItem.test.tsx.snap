// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconItem IconItemを描写 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fffffe",
      "borderRadius": 10,
      "justifyContent": "center",
      "margin": 4,
      "opacity": 1,
      "padding": 10,
      "width": 120,
    }
  }
>
  <Text />
  <Text
    style={
      Object {
        "color": "#333",
        "fontSize": 14,
        "margin": 4,
      }
    }
  >
    star
  </Text>
</View>
`;
